source/AI.cpp

```diff
@@ -1918,7 +1918,10 @@ bool AI::MoveTo(Ship &ship, Command &command, const Point &targetPosition,
 	bool isFacing = (dp.Unit().Dot(angle.Unit()) > .95);
 	if(!isClose || (!isFacing && !shouldReverse))
 		command.SetTurn(TurnToward(ship, dp));
-	if(isFacing)
+	// Work with a slightly lower maximum velocity to avoid border cases.
+	double maxVelocity = ship.MaxVelocity() * ship.MaxVelocity() * .99;
+	if(isFacing && (ship.Velocity().LengthSquared() <= maxVelocity
```

Comment:
```suggestion
	if(isFacing && (velocity.LengthSquared() <= maxVelocity
```

Looks like velocity was already called at the top of the function.  Not sure if this matters.

Reply:
good catch